components/monitoring/logging/base/configure-logging.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
+metadata:
+  name: "instance"
+  namespace: "openshift-logging"
+spec:
+  collection:
+    logs:
+      fluentd: {}
+      type: fluentd
+---
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  outputs:
+    - cloudwatch:
+        groupBy: logType
```

Comment:
I think it would be better to use `namespaceName` so we will also group logs by their origin namespace.

Reply:
yeah. but I believe we are more interested on the application logs. As audit and infra logs would be available from sre @amfred how do you think on this?

Reply:
I see that you are also forwarding audit an infra logs. What do you mean that it will be available from sre?

Reply:
yes, My plan is remove the audit and infra logforwarding once we know we can access the audit logs from sre. may be lets make it to groupby `namespaceName` to make it similar to the host cluster log forwarding setup by sre team.

Reply:
SRE doesn't have any connection to our member clusters, so I don't think we can count on them for infra and audit logs.